packages/kit/src/views/Market/MarketDetailV2/components/TokenDetailHeader/TokenDetailHeader.tsx

```
@@ -1,69 +1,202 @@
+    if (website) {
+      openUrlExternal(website);
+    }
+  };
+
+  const handleOpenTwitter = () => {
+    if (twitter) {
+      openUrlExternal(twitter);
+    }
+  };
+
+  const priceChangeNum = parseFloat(priceChange24hPercent);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)